ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -24,6 +25,25 @@ class ModelUtils:
         EncoderType.RESNET: 15,
     }
 
+    VALID_VISUAL_PROP = frozenset(
+        [
+            (
```

Comment:
Can we change one of the existing sensors (Camera, RenderTexture, Match3, or Grid) to use `(TRANSLATIONAL_EQUIVARIANCE, TRANSLATIONAL_EQUIVARIANCE, NONE)` to make sure we exercise this path?